lib/components/user/favorite-locations-pane.js

```diff
@@ -1,8 +1,8 @@
-import clone from 'lodash/cloneDeep'
+import { Field, FieldArray } from 'formik'
```

Comment:
I'm just realizing that the amount of files in the `lib/components/user` directory is getting quite large. Can some subfolders be created to help organize all these files?

Reply:
Good point. Perhaps this? In this PR or a separate PR?

user/
- <generic UI components> (name needed)
  * form-navigation-buttons
  * link-button`
  * link-menu-item
  * stacked-pane-display
  * sequential-pane-display
- login
  * after-signin-screen
  * awaiting-screen
  * before-signin-screen
  * nav-login-button
  * nav-login-button-auth0
  * nav-login-button.css
  * with-loggedin-user-support
- account
  * account-setup-finish-pane.js
  * existing-account-display.js
  * favorite-locations-pane.js
  * new-account-wizard.js
  * notification-prefs-pane.js
  * phone-verification-pane.js
  * terms-of-use-pane.js
  * user-account-screen.js
  * verify-email-screen.js
- trip
  * saved-trip-editor.js
  * saved-trip-list.js
  * saved-trip-screen.js
  * trip-basics-pane.js
  * trip-notifications-pane.js
  * trip-summary-pane.js
  * trip-summary.js

Reply:
Might as well do a separate PR. For the `(name needed)` I'd just keep those in the root `/user` folder.